RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@
 18.0
 -----
 * [*] Need a little help after login? We're here for you. We've made a few changes to the login flow that will make it easier for you to start managing your site or create a new one.
+* [*] Unified Comments: Replaced comment list with a new version [https://github.com/wordpress-mobile/WordPress-Android/pull/15107/]
```

Comment:
np: maybe I would place this a bit more in context like "Unified Comments: as part of the comments experience unification effort we have replaced filtered comment lists with a new version"

Reply:
Updated, thank you for the suggestion!